htdocs/fourn/commande/list.php

```diff
@@ -291,7 +293,7 @@
 				$objecttmp->fk_project = $cmd->fk_project;
 				$objecttmp->multicurrency_code = $cmd->multicurrency_code;
 				if (empty($createbills_onebythird)) {
-					$objecttmp->ref_client = $cmd->ref_client;
+					$objecttmp->ref_supplier = !empty($cmd->ref_supplier) ? $cmd->ref_supplier : $default_ref_supplier+1;
```

Comment:
This will works only for 2 values.
The 3rd one will have  $default_ref_supplier+1; that will be same than the 2nd one.

Reply:
@eldy of course, sorry i probably was asleep while coding this morning...